io/shared/src/main/scala/fs2/io/process/Process.scala

```diff
@@ -34,11 +34,14 @@ sealed trait Process[F[_]] {
 
   /** A `Pipe` that writes to `stdin` of the process. The resulting stream should be compiled
     * at most once, and interrupting or otherwise canceling a write-in-progress may kill the process.
+    * Failure to write promptly to this `Stream` when required can may cause the process to block, or
```

Comment:
```suggestion
    * Failure to write promptly to and/or close this `Stream` when required can may cause the process to block, or
```

Maybe there's a better way to say it. But in some cases you may need to explicitly write an empty stream to close it.

https://github.com/typelevel/fs2/blob/d3c6900a33e44544a2ff1228605cf7b899e1b00f/io/shared/src/test/scala/fs2/io/process/ProcessSuite.scala#L183-L189